### PR TITLE
[Product Partnerships] Added Amazon WorkMail

### PR DESCRIPTION
**Pull request by rahul188**
# Summary

A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this pull request is ready for review and merge. **If it is not ready, please mark the pull request as a draft.**

_The owners of this repo are not experts in the subject matter of the quickstarts. We review for the quickstart to be functional and for security risks. If you are seeking feedback on the content of the quickstart, please seek out a subject matter expert. If you are not an internal NR contributor, we can do our best to connect you with a content reviewer._

## Pre merge checklist

- [x] Did you check you NRQL syntax? - Does it work?
- [X] Did you include a Data source and Documentation reference?
- [X] Are all documentation links publicly accessible?
- [X] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [X] Did you check your descriptive content for spelling and grammar errors?
- [X] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [X] Does the PR contain a screenshot for each of your dashboards?
- [X] Do your screenshots show data?
- [X] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [X] Did you check that your alerts actually work?

### CODE REVIEW

File: quickstarts/aws/amazon-workmail/config.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      Monitor Amazon WorkMail by connecting AWS to New Relic.
+    url: >-
+      https://docs.newrelic.com/docs/infrastructure/amazon-integrations/aws-integrations-list/amazon-workmail-monitoring-integration/
+keywords:
+  - aws
+  - amazon web services
+  - WorkMail
+dashboards:
+  - amazon-workmail
+dataSourceIds:
```
Comment (brammerl) [inline quickstarts/aws/amazon-workmail/config.yml:28]: you're not referencing the `amazon-workmail` data source. did you want to reference it? If you're not using the data source you created please remove it.

**Review by brammerl — CHANGES_REQUESTED** [review]

File: quickstarts/aws/amazon-workmail/config.yml (hunk 1 above)
Comment (rahul188) [reply quickstarts/aws/amazon-workmail/config.yml:28]: @brammerl , Addressed, I made the required changes.

File: alert-policies/amazon-workmail/IncomingEmailBounced.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+name: High IncomingEmailBounced
```
Comment (josephgregoryii) [inline alert-policies/amazon-workmail/IncomingEmailBounced.yml:1]:
```suggestion
name: High Incoming Email Bounced
```

File: alert-policies/amazon-workmail/IncomingEmailBounced.yml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+description: |+
+  This alert is triggered when the IncomingEmailBounced is above 100 in 10 minutes.
```
Comment (josephgregoryii) [inline alert-policies/amazon-workmail/IncomingEmailBounced.yml:4]:
```suggestion
  This alert is triggered when the Incoming Email Bounced is above 100 in 10 minutes.
```

**Review by josephgregoryii — CHANGES_REQUESTED** [review]
Looks like we're forgetting some spaces in the alert names

File: alert-policies/amazon-workmail/IncomingEmailBounced.yml (hunk 3 above)
Comment (rahul188) [reply alert-policies/amazon-workmail/IncomingEmailBounced.yml]: @josephgregoryii , Addressed the changes

File: alert-policies/amazon-workmail/IncomingEmailBounced.yml (hunk 2 above)
Comment (rahul188) [reply alert-policies/amazon-workmail/IncomingEmailBounced.yml]: @josephgregoryii , Addressed the changes

**Review by d3caf — COMMENTED** [review]
Have a wording changes but overall, looks good! Thanks!

**Review by brammerl — DISMISSED** [review]

**Comment by brammerl** [comment]
Ignore my approval. I couldn't dismiss the review but clicked approve instead.

**Review by josephgregoryii — DISMISSED** [review]

File: alert-policies/amazon-workmail/IncomingEmailBounced.yml [hunk 4]
```diff
@@ -0,0 +1,27 @@
+name: High Incoming Email Bounced
```
[the remark this answers is not in the document]
Comment (rahul188) [reply alert-policies/amazon-workmail/IncomingEmailBounced.yml]: @d3caf, Addressed Required Changes.

File: alert-policies/amazon-workmail/IncomingEmailBounced.yml [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+description: |+
+  This alert is triggered when the Incoming Email Bounced is above 100 in 10 minutes.
```
[the remark this answers is not in the document]
Comment (rahul188) [reply alert-policies/amazon-workmail/IncomingEmailBounced.yml]: @d3caf, Addressed Required Changes.

File: alert-policies/amazon-workmail/OutgoingEmailBounced.yml [hunk 6]
```diff
@@ -0,0 +1,27 @@
+name: High Outgoing Email Bounced
```
[the remark this answers is not in the document]
Comment (rahul188) [reply alert-policies/amazon-workmail/OutgoingEmailBounced.yml]: @d3caf, Addressed Required Changes.

File: alert-policies/amazon-workmail/OutgoingEmailBounced.yml [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+
+description: |+
+  This alert is triggered when the Outgoing Email Bounced is above 100 in 10 minutes.
```
[the remark this answers is not in the document]
Comment (rahul188) [reply alert-policies/amazon-workmail/OutgoingEmailBounced.yml]: @d3caf, Addressed Required Changes.

**Review by mickeyryan42 — APPROVED** [review]

**Pull request merged**